testing/InstallTestbed.ps1

```diff
@@ -0,0 +1,309 @@
+param (
+    [Parameter(Mandatory = $true)]
+    [string]$ResourceGroup,
+
+    [string]$VMUsername = "admin.ackbar",
+    [string]$VMName = "DC1",
+    [string]$LinuxVMName = "LS1",
+    [int]$NumberOfClients = 2
+)
+
+# If you were to need the password from the SetupTestbed.ps1 script, you could use this:
+# $Password = Get-Content "password.txt"
+
+# Define our library path
+$libraryPath = Join-Path -Path $PSScriptRoot -ChildPath "configure\azure_scripts\lib\utilityFunctions.ps1"
```

Comment:
You're generally pretty good about this, but I'd recommend changing this to `$LibraryPath` for consistency, as PowerShell generally is written in PascalCase.